flashinfer/jit/gdn.py

```
@@ -0,0 +1,37 @@
+
+def gen_gdn_prefill_sm90_module() -> JitSpec:
+    return gen_jit_spec(
+        name="gdn_prefill_launcher",
+        sources=[
+            jit_env.FLASHINFER_CSRC_DIR / "gdn_prefill_launcher.cu",
+            jit_env.FLASHINFER_CSRC_DIR
+            / "flat"
+            / "prefill"
+            / "prefill_kernel_delta_rule_sm90.cu",
+        ],
+        extra_cuda_cflags=sm90a_nvcc_flags + ["-DFLAT_SM90A_ENABLED", "-std=c++20"],
```

Ruff (0.14.10)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)